app/views/questions/new.html.erb

```diff
@@ -5,23 +5,74 @@
         <div class="box">
           <h2 class="title-no-margin center">New Question</h2>
           <br>
-            <%= form_for @question do |f| %>
-              <div class="form-group">
-                <%= f.text_field :text, placeholder: 'Question', class: 'form-control', autofocus: true %>
-              </div>
-              <div class="form-group">
-                <%= f.label :qset_id, 'Qset', {class: 'control-label'} %>
-                <%= f.collection_select(:qset_id, Qset.find(current_user.org).descendants, :id, :name, {}, {class: 'form-control'}) %>
+            <%= form_for(@question) do |f| %>
+              <%= f.text_field :text, placeholder: 'Question',  class: 'form-control', autofocus: true %>
+              <button type="button" class="btn btn-primary btn-theme btn-generate-q" data-toggle="modal" data-target="#myModal">
+                Choose Qset
+              </button>
+              <span class="qset-display">
+                <% if !params[:qset].nil? %>
+                  <%= @qsets.find(params[:qset]).name %>
+                <% end %>
+              </span>
+              <!-- hidden input field to pass in qset_id into params on submit -->
+              <% if !params[:qset].nil? %>
+                <input type="text" name="question[qset_id]" id="qset_id_input" class="hide-display" value="<%= @qsets.find(params[:qset]).id %>">
```

Comment:
Typically IDs and classes use dashes instead of underscores. The reason so many of our IDs use underscores is (a) Rails form helpers auto-generate IDs that use underscores, and (b) we may have let our Rails coding style slip into our HTML/JS/CSS coding styles. If possible, let's try to keep any manual work we do consistent with best practices.

https://google.github.io/styleguide/htmlcssguide.xml?showone=ID_and_Class_Name_Delimiters#ID_and_Class_Name_Delimiters